crates/core/src/sql/compiler.rs

```diff
@@ -135,18 +135,15 @@ fn compile_columns(table: &TableSchema, cols: &[ColId]) -> DbTable {
     let mut columns = Vec::with_capacity(cols.len());
     let cols = cols
         .iter()
+        // TODO: should we error here instead?
+        // When would the user be passing in columns that aren't present?
         .filter_map(|col| table.get_column(col.idx()))
```

Comment:
This will be gone when the new `sql parser` lands, and it will do the proper checkings.